src/tasks.ts

```diff
@@ -44,7 +44,16 @@ async function provideJuliaTasksForFolder(folder: vscode.WorkspaceFolder): Promi
             let testTask = new vscode.Task({ type: 'julia', command: 'test' }, folder, `Run tests`, 'julia', new vscode.ProcessExecution(g_settings.juliaExePath, ['--color=yes', '-e', 'Pkg.test(Base.ARGS[1])', rootPath]), "");
             testTask.group = vscode.TaskGroup.Test;
             testTask.presentationOptions = { echo: false };
-            result.push(testTask);
+            result.push(testTask);   
+        }
+
+        if (await fs.exists(path.join(rootPath, 'REQUIRE'))) {
+            let scriptpath = path.join(g_context.extensionPath, 'scripts', 'test_deps.jl')
+            let depTestTask = new vscode.Task({ type: 'julia', command: 'test' }, folder, `Run tests on dependents`, 'julia', new vscode.ProcessExecution(g_settings.juliaExePath, ['--color=yes', '-e', 'include("' + scriptpath + '")', path.basename(rootPath)]), "");
```

Comment:
I believe we need to use a different string for ``command`` because ``test`` is used above already. Maybe just ``testdependencies``?

Can we make the description ``Run tests of dependent packages``? Or something like that?

I think we can skip the whole ``-e 'include(...)'`` stuff and just pass ``scriptpath`` as the second element in that array?